src/Nordea/Components/MultiSelectDropdown.elm

```diff
@@ -147,7 +147,12 @@ view attrs dropdown =
                                 dropdown.onFocus (not dropdown.hasFocus)
 
                             Events.Space ->
-                                dropdown.onFocus (not dropdown.hasFocus)
+                                case dropdown.inputProperties of
+                                    Just inputProperties ->
+                                        inputProperties.onInput (inputProperties.input ++ " ")
```

Comment:
For accessibility with a dropdown, you can tab to it to make it active, but doing so does not expand it right away. Clicking space expands the dropdown allowing you to choose something or press esc to close it and move to the next input. 

How does this behaviour work with input?

Reply:
it currently doesnt really work

should be possible i guess, but i dont really dont know. seems hard with my UU knowledge
We can maybe try fixing it in a session?